src/calendar/day/index.tsx

```diff
@@ -37,7 +47,7 @@ export default class Day extends Component {
     ]);
   }
 
-  getMarkingLabel(marking) {
+  getMarkingLabel(marking: any) {
```

Comment:
Why not object?

Reply:
Actually setting it to `object` is almost like setting it to `any` if we don't write the object keys.
And I don't really know the possible keys for `marking`

Reply:
I disagree. 'object' is more restrictive than 'any' type and indicates that the value is a non-primitive.